layouts/partials/newsletterForm.html

```diff
@@ -0,0 +1,29 @@
+<div class="bg-[#1D65A6] max-w-[800px] mx-auto mt-20 py-14 px-12 md:px-16 text-white md:rounded-lg">
```

Comment:
The newsletter form code was moved to another file, but no other changes were made.